HerPortal.ManagementShell/DatabaseOperation.cs

```diff
@@ -17,21 +17,34 @@ public DatabaseOperation(HerDbContext dbContext, OutputProvider outputProvider)
         this.outputProvider = outputProvider;
     }
 
-    public List<User> GetUsersWithLocalAuthorities()
+    public List<User> GetUsersWithLocalAuthoritiesAndConsortia()
     {
         return dbContext.Users
             .Include(user => user.LocalAuthorities)
+            .Include(user => user.Consortia)
             .ToList();
     }
 
-    public List<LocalAuthority> GetLas(string[] custodianCodes)
+    public List<LocalAuthority>? GetLas(IReadOnlyCollection<string> custodianCodes)
     {
+        if (custodianCodes.Any(code => !dbContext.LocalAuthorities.Any(la => la.CustodianCode == code))) return null;
```

Comment:
I'd be a bit surprised if I came across this: this feels like a case for throwing an error rather than returning `null`. If I've got a method like `GetSomeListOfThings`, I'll expect it to return a list (that may or may not be empty) - returning `null` doesn't have any intuitive meaning.

You'll want a custom exception that you can throw and catch upstairs. If you do that you can attach the problematic code(s) as a property on the exception and then refer back to them to log in the output. You might find that doing this makes your code flow a little better too, as you can just catch the exception as it bubbles up rather than having to immediately check for `null`